cityofaustin/atd-vz-data
# 9519 edit recommendations

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/9519

This PR makes Fatality Board Review Recommendations:
- editable
- only viewable by those with admin or IT Supervisor roles
- only apply to crashes with `atd_fatality_count > 0`

## Testing
**URL to test:** 
https://deploy-preview-1091--atd-vze-staging.netlify.app/#/crashes/

**Steps to test:**
1. Go to a crash details page that doesn't already have recommendation details
2. To see the recommendations card, you must:
  - Be logged in with an account that is an admin or IT Supervisor
  - Be looking at a crash with `atd_fatality_count` > 0 (you can update this field to switch the card on and off)
3. To create the initial record, you can:
  - Select a **Coordination Partner** from the dropdown
  - Select a **Status** from the dropdown
  - Click "Add" next to the **Recommendation** field and click the checkbox to save
  - Click "Add" next to the **Updates** field and click the checkbox to save
4. After one of these four actions, the recommendation record exists and further updates will be edits to the existing record
5. Test updating the values in any of the fields and you will see the value persist
6. Test the cancel button when editing either **Recommendation** or **Updates** to make sure that the value is not edited even if you update it in the text area input

---
#### Ship list
